client/src/layout/TabletMenu/TabletMenu.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { NavLink } from 'react-router-dom';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import styles from './TabletMenu.module.scss';
+
+const pages = [
+  {
+    name: 'Home',
+    link: '/',
+    innerContent: 'Home',
+  },
+  {
+    name: 'Catalogue',
+    link: '/catalogue',
+    innerContent: 'Catalogue',
+  },
+  {
+    name: 'Cart',
+    link: '/cart',
+    innerContent: <ShoppingCartIcon fontSize="medium" />,
+  },
+];
+
+const TubletMenu = () => {
+  const boxSX = {
+    flexGrow: 1,
+    display: { sx: 'none', laptop: 'flex' },
```

Comment:
please change to the below as i edited the theme,  the sx value was not working and the tablet menu was not getting display: none when viewing on mobile.

`display: { zero: 'none', laptop: 'flex' },`

Reply:
done